tests/dogfood.rs

```diff
@@ -23,7 +32,7 @@ fn dogfood_clippy() {
 
     // "" is the root package
     for package in [
-        "",
```

Comment:
I'm not sure if this is new behavior but, now, this step also seems to check `clippy_lints` and `clippy_utils`. At least it says so in the output. Could you investigate if lints can be emitted here? And also, we could see if there is a way to merge these steps.

Reply:
The `""` -> `"./"` change is just to make the `println` clearer, it shouldn't change any behaviour

I don't think there's a good way unless we do some driver crimes, when it's compiling clippy_lints/clippy_utils here it's using rustc because we're not in a workspace

I *think* we reordered it at some point and the root package first is as good as it gets since it will compile the most dependencies in parallel